src/Components/ElementsPanel/ViewerElementsPanel.tsx

```diff
@@ -24,6 +31,28 @@ const ViewerElementsPanel: React.FC<IViewerElementsPanelProps> = ({
     const [filterTerm, setFilterTerm] = useState('');
     const nodeRef = React.useRef(null); // <Draggable> requires an explicit ref to avoid using findDOMNode
 
+    const getVisualRulesIncludingFilterTerm = (panelItem) => {
+        // Map all visuals from existing behaviors
+        const visuals = [].concat(...panelItem.behaviors.map((b) => b.visuals));
```

Comment:
why not just: 
```suggestion
        const visuals = [...panelItem.behaviors.map((b) => b.visuals)];
```

or  const visuals = panelItem.behaviors.map((b) => deepCopy(b.visuals))